lib/puppet/ssl/certificate_authority.rb

```diff
@@ -121,7 +121,11 @@ def generate(name, options = {})
     raise ArgumentError, _("A Certificate already exists for %{name}") % { name: name } if Puppet::SSL::Certificate.indirection.find(name)
 
     # Pass on any requested subjectAltName field.
-    san = options[:dns_alt_names]
+    if options[:dns_alt_names] && options[:subject_alt_names]
+      san = options[:dns_alt_names] + ", " + options[:subject_alt_names]
+    else
+      san = options[:dns_alt_names] || options[:subject_alt_names]
+    end
```

Comment:
@ahpook do we want to deprecate (i.e. actually issue a deprecation warning) the old flag? Or do we intend to maintain both going forward?